FAILURE\
a panicking function std::process::abort is invoked
SUCCESS\
a panicking function std::process::abort is invoked
